dataactbroker/README.md

```diff
@@ -555,14 +555,12 @@ Example output:
 
 ## Test Cases
 
-To run the broker API unit tests, navigate to the main project folder (`data-act-broker`) and type the following:
+To run the broker API unit tests, navigate to the main project folder (`data-act-broker-backend`) and type the following:
```

Comment:
Could we just change these directions to instruct people to be in the tests folder rather than the main project folder?  Is there a benefit to running them from there?